backup-restore/uninstall/uninstall-backup-restore.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash 
+# Performs a full uninstall on Backup and Restore from the cli.
```

Comment:
Make comment more clear: Fusion Backup & Restore service, this is not Legacy (at least I don't think so)

Reply:
Done.